delete-user-data/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.9
+
+feature - added warsaw (europe-central2) location (#677)
```

Comment:
nit - consider capitalizing Warsaw in each changelog

Reply:
Done.